src/app/framework-modules/elements/ct-field-generator/component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, Input } from '@angular/core';
+import { FormControl} from '@angular/forms';
+import { Form } from '../../formdata/form/form';
+import { FormData } from '../../formdata/form/formData';
+
+@Component({
+	selector: 'ct-field-generator',
+	templateUrl: './component.html',
+	styleUrls: []
+})
+
+/**
+ * Automated field Generator component
+ * @fd - Formdata
+ */
+export class CTFieldGenerator {
+	@Input() public fd: FormData;
+
+	fields:any;
+	form:Form;
+	control:FormControl;
+	
+	ngOnInit() {

```

Comment:
Another mysterious abbreviation.  What is `ng`?  Next generation?:). And why "OnInit"?  Is that an angular thing?

Could this function use a document comment and/or a more self-documenting name?

Reply:
ngOnint is an in-built angular lifecycle hook - https://angular.io/api/core/OnInit